Publisher/Sources/Services/DefaultAblyPublisherService.swift

```diff
@@ -18,13 +18,15 @@ class DefaultAblyPublisherService: AblyPublisherService {
 
     private func setup() {
         client.connection.on { [weak self] stateChange in
-            guard let current = stateChange?.current,
-                  let self = self
-            else { return }
-            logger.debug("Connection to Ably changed. New state: \(current)", source: "DefaultAblyPublisherService")
+            guard let self = self,
+                  let receivedConnectionState = stateChange?.current.toConnectionState() else {
```

Comment:
`toConnectionState` is a method from an extension? It converts Ably connection state to the project state? Is that it?

Reply:
Yes, you are right. 

```
extension ARTRealtimeConnectionState {
    func toConnectionState() -> ConnectionState {
        ...
    }
}
```